src/grid.scss

```diff
@@ -28,26 +37,53 @@
  *                          --gd-cols-cus-xl  (num of cus cols @xl )
  *                          --gd-cols-cus-2xl (num of cus cols @2xl )
  *
- *                        [Modifier 1: gap]
- *                          --gd-gap-cus     ( custom gap size )
- *                          --gd-gap-cus-m   ( gap size m )
- *                          --gd-gap-cus-l   ( gap size l )
- *                          --gd-gap-cus-xl  ( gap size xl )
- *                          --gd-gap-cus-2xl ( gap size 2xl )
- *
- *                    subgrid (subgrid within col)
+ *                    subgrid (subgrid within a col)
  *
  *                      Variables used:
  *                        [Modifier 1: col]
- *                          --gd-col-cus ( for cus subgrid within a col )
+ *                          --gd-col-cus      ( for cus subgrid within a col )
  *
+ *                      Variables used:
+ *                        [Modifier 1: gap]
+ *                          --gd-gap-cus      ( for cus gap size )
+ *                          --gd-gap-cus-col  ( to extract the cus col size)
  */
 
 .gd-gap-cus { gap: var(--gd-gap-cus, repeat( $gd-max-cols, 1fr)); }
-.gd-gap-cus-m { gap: $gd-gap-s; }
-.gd-gap-cus-l { gap: $gd-gap-m; }
-.gd-gap-cus-xl { gap: $gd-gap-l; }
-.gd-gap-cus-2xl { gap: $gd-gap-xl; }
+.gd-gap-s { gap: $gd-gap-s; }
+.gd-gap-m { gap: $gd-gap-m; }
+.gd-gap-l { gap: $gd-gap-l; }
+.gd-gap-xl { gap: $gd-gap-xl; }
+
+.gd-line{
```

Comment:
We can use the class name as dvd - meaning divider. Line can imply anything. Divider is the common notation used elsewhere.